src/ngraph/runtime/interpreter/int_executable.hpp

```diff
@@ -831,6 +832,18 @@ class INTERPRETER_BACKEND_API ngraph::runtime::interpreter::INTExecutable : publ
                                             max_pool_backprop->get_padding_above());
             break;
         }
+        case OP_TYPEID::Mod_v1:
```

Comment:
Why do it here instead of with an evaluate method?

Reply:
Since the OV change already finished, not sure if there is still a need to do the evaluate method. I can add the evaluate to the op.

Reply:
Use the evaluate. More things can take advantage of it that way, like running with dynamic shapes.